bot/controllers/viewPoints.js

```diff
@@ -0,0 +1,38 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+const {
+  Member,
+} = require('../../db/models');
+
+class ViewPointsController extends BaseController {
+  constructor(message) {
+    super(message);
+    const controller = this;
+    this.commands = [
+      new Command(
+        '!viewPoints',
+        '!viewPoints',
+        'Return Users Current Points',
+        'Receive a message with the users current points.',
+        this.viewPointsAction.bind(controller),
+        'dm',
+      ),
+    ];
+  }
+
+  async viewPointsAction() {
+    const { message } = this;
+    // Get Member Data
+    const memberData = await Member.findAll({
+      attributes: ['points'],
+      where: {
+        discordUser: message.author.id,
+      },
+    });
+    // Get points
+    const points = memberData[0].dataValues.points;
+    message.author.send(`Your total Points: ${points}`);
```

Comment:
I just want to make sure, is this because of the issue getting data from the promises? I know I ran into this same issue where I can't use return, so I'm just making sure that is the reason.

Reply:
yeah I was not able to use return

Reply:
Makes sense, there's a lot of issues with promises.